src/tokens/transformation/android/filters/lightColor.js

```diff
@@ -0,0 +1,4 @@
+const { matchLightThemeToken } = require('../../common/tokenFilters')
+
+module.exports = (token) =>
+  matchLightThemeToken(token) || token.name.startsWith('color_primitive')
```

Comment:
I think maybe we should have three sets of tokens
1. Light
2. Dark
3. Primitive